primitiv/__init__.py

```diff
@@ -1,3 +1,51 @@
+def _load_libprimitiv():
```

Comment:
Loading procedure for required binaries should not be written by hand. This is like an OS's work.